### DISCUSSION TITLE
Menu

### DISCUSSION THREAD

**Discussion by Podberezovik34** [smplayer-dev/smplayer#1059] [Q&A]
Good afternoon! Please tell me how to remove the top menu from the playback screen? I found how to remove the main panel, but I can't find how to remove this menu.

**Comment by Podberezovik34** [comment]
![IMG_20241003_153049_123](https://github.com/user-attachments/assets/09c0787c-faa3-440f-a91a-385b6b33092f)
Here it is, in the screenshot. It just gets in the way, considering that the same menu can be called with the right mouse button in the form of a context menu.

**Comment by smplayer-dev** [comment]
Ctrl-C hides the menu (and other elements).
it's not possible to hide the menu only.
